testing/smoke/lib.sh

```diff
@@ -181,7 +181,7 @@ healthcheck() {
     RES=$(curl_fail -H "${APM_AUTH_HEADER}" ${APM_SERVER_URL}) || RC=$?
     if [ $RC -ne 0 ]; then echo "${RES}"; fi
     local PUBLISH_READY=$(echo ${RES}| jq '.publish_ready')
-    if [[ ! ${PUBLISH_READY} ]]; then
+    if [[ ${PUBLISH_READY} != true ]]; then
```

Comment:
I would suggest to explicitly use string comparison here:

```suggestion
    if [[ "${PUBLISH_READY}" != "true" ]]; then
```

In practice this behavior should not be affected, but it leaves the door open to nasty bugs (there is an interesting answer on StackOverflow detailing [some](https://stackoverflow.com/a/21210966)) especially if the server response changes at some point.

Reply:
even better!

Reply:
Thanks!